pkg/kubectl/scale.go

```diff
@@ -87,30 +87,6 @@ func (pe PreconditionError) Error() string {
 	return fmt.Sprintf("Expected %s to be %s, was %s", pe.Precondition, pe.ExpectedValue, pe.ActualValue)
 }
 
-type ScaleErrorType int
-
-const (
-	ScaleGetFailure ScaleErrorType = iota
-	ScaleUpdateFailure
-	ScaleUpdateConflictFailure
-)
-
-// A ScaleError is returned when a scale request passes
-// preconditions but fails to actually scale the controller.
-type ScaleError struct {
```

Comment:
I don't think that this wrapper added value, since we only special cased conflicts, which are easily identifiable.

Reply:
you are absolutely right.